lib/WUI/wui_api.h

```diff
@@ -186,16 +189,16 @@ uint8_t dhcp_addrs_are_supplied(void);
 /*!****************************************************************************
 * \brief Parses time from device's time storage in dest string in format hh:mm:ss
 *
-* \param dest - destination string for paresd time ! At least 12 chars !
+* \param system_time - destination string for paresd time, wrapped as 12 chars string
```

Comment:
Typo: paresd -> parsed
wrapped as 12 chars - Redundant. It is clear from parameter type, what is expected here. It is impossible to pass something else. Also if someone change number to 18, it is highly probable, that he will forget to update this comment  and it becomes obsolete.
string - Not very specific. What kind of string? If it is null-terminated string, it would be worth to note, it is null-terminated string (C-style string).
You can also use doxygen \param direction to make it clear, it is output value of function. e.g.:
```
* \param [out] system_time Parsed time (null-terminated string)
```

Reply:
It would be also worth to note here that returns "N/A" in some case.

Reply:
If you consider this construct non-trivial, better than write that something is wrapped somewhere (no-one would understand what it means), you can add usage example in doxygen documentation e.g.:
```
* Usage:
* \code
* system_time_t current_time;
* sntp_get_system_time(&current_time);
* printf(current_time); 
* \endcode
```

Reply:
commit `e98f5b1`